README.md

```diff
@@ -154,3 +154,32 @@ Report bugs & give feedback
 If you run into any bugs or have feedback you'd like to share, please let us know on GitHub.
 
 To help avoid duplicate issues, please [view existing issues](https://github.com/vector-im/element-web/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc) first (and add a +1) or [create a new issue](https://github.com/vector-im/element-web/issues/new/choose) if you can't find it.  Please note that this issue tracker is associated with the [element-web](https://github.com/vector-im/element-web) repo, but is also applied to the code in this repo as well.
+
+
+Trouble Shooting
+==========================
+
+Sometimes there is Cache from an old Riot folder or an old Element folder:
```

Comment:
I think this needs to say a bit more about why someone would care. When should they try these steps?